libs/common/src/response-messages/index.ts

```diff
@@ -346,7 +346,8 @@ export const ResponseMessages = {
             platformConfigNotFound: 'Platform config not found',
             batchEmailSend: 'Unable to send email in batches',
             emailSend: 'Unable to send email to the user',
-            verificationRecordsNotFound: 'Verification records not found'
+            verificationRecordsNotFound: 'Verification records does not exists',
+            removeVerificationData: ', first you have to remove verification data'
```

Comment:
Is the ',' required in the error message?

Reply:
Resolved